src/app/clusters/ota-requestor/ota-requestor-server.cpp

```diff
@@ -122,6 +127,22 @@ EmberAfStatus OtaRequestorServerSetUpdateStateProgress(uint8_t value)
     return status;
 }
 
+EmberAfStatus OtaRequestorServerGetUpdateStateProgress(chip::EndpointId endpointId, uint8_t & value)
```

Comment:
`value` should be a Nullable assuming you'd want to differentiate between 0 vs null